fees-rewards.md

```diff
@@ -0,0 +1,75 @@
+# Fees and Rewards
+
+Obscuro sets out a simple structure for fees and rewards. End users pay fees based on their usage which are placed into a pool from which rewards for nodes are paid out.
+
+The structure delivers a steady and predictable income for node operators over the long term and a predictable fee for users proportional to the actual costs of running the network and does not rely on bidding wars or vary according to activity.
+
+The structure also does not punish Aggregators that miss out on a publishing opportunity with slashing. Instead, the loss of reward should be incentive enough.
+
+## Cost of L2 nodes
+
+Operators run the network and incur operational costs by running and maintaining nodes.
+
+An "expected monthly node operational costs" variable is set through the governance protocol, which is designed to reflect how much a node operator would pay for hardware, hosting etc, in a month. This is analogous to the proof of work difficulty found in other blockchains. This effects demand for running a node.
+
+Aggregator nodes incur cost from submitting transactions to L1, through the following:
+
+- Storing rollup data on Ethereum.
+- The overhead of processing the L1 transaction that contains the rollup, e.g. signing transactions.
+
+## Fee per transaction
+
+The total fee a user must pay has three components.
+
+1. A proportion of the monthly operational cost for all nodes. The proportion is calculated based on the computation of the current transaction as a proportion of the moving average of all L2 transactions over the month. Represented as:
+
+![proportion_equation.png](images/equation_1.png)
+
+![operation_equation.png](images/equation_2.png)
+
+2. The cost incurred by nodes for L1 storage, represented as:
+
+![storage_equation.png](images/equation_3.png)
+
+3. A proportion of the additional overhead cost incurred by nodes for processing rollups is as follows:
+
+The idea behind this is:
+
+- At the point of transaction, it is impossible to know what the true cost of the future rollup is
+- Users need to pay their fair share of what this future rollup cost will be
+- The best indicator of what the future rollup cost might be is a moving average of the previous n rollups
+- Where n is high, the averaging is performed over a larger number of rollups and therefore less volatile and vice versa. High n adds more risk to Aggregators
```

Comment:
Let's remove : High n adds more risk to Aggregators
because it's unclear how it works